pkgs/applications/misc/smcipmitool/default.nix

```
@@ -0,0 +1,68 @@
+{ lib, stdenv
+, fetchurl
+, makeWrapper
+, patchelf
+, fontconfig
+, freetype
+, gcc
+, gcc-unwrapped
+, iputils
+, psmisc
+, xorg }:
```

```suggestion
}:
```

Call the xorg sub-packages directly.